components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -269,7 +274,7 @@ export const BrandedWallpaperNotification = styled('div')`
 `
 
 export const GridItemNavigation = styled('section')`
-  grid-column: 2 / span 2;
+  grid-column: 3;
```

Comment:
This item occupied 2 columns and made, which is not needed because it has only one child.